com.github.PopoutApps.popout3d.yml

```
@@ -0,0 +1,173 @@
+app-id: com.github.PopoutApps.popout3d
+
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+
+command: popout3d
+rename-desktop-file: popout3d.desktop
+rename-icon: popout3d_logo
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host:rw
```

```suggestion
  - --filesystem=host
```

This is the default.

---

Done.